evap/static/tests/test_live.py

```diff
@@ -0,0 +1,49 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.expected_conditions import visibility_of_element_located
+
+from evap.evaluation.tests.tools import LiveServerTest
+
+
+class NoteBookPlacementLiveTest(LiveServerTest):
```

Comment:
@niklasmohrin what's your opinion on these tests? So far, most of our frontend tests test/assert things that are stable, consistent user-visible truths ("There should be a button "Submit", click it, the page should go to XYZ, a success message with text "ABC" should be visible somewhere on the page"). Asserting stuff like "At 1400px browser window width, the element should have a width of 25vw" seems a bit too much like a meaningless/brittle implementation detail to me. It does not assert in a meaningful way that the page layout is sane, and it likely will fail exactly when people intentionally change the CSS.

I'm currently don't think this adds enough much value that it justifies the test runtime (how long does each of these tests take, 500ms?)